apps/graphql/src/apps/booking/queries/SaveBooking.js

```diff
@@ -12,7 +12,7 @@ type Args = {|
 const SaveBooking = {
   name: 'SaveBooking',
   description:
-    'Mutation (POST Request) that saves a given booking. Initiates the reservation process on the backend.',
+    'Saves a given booking. Initiates the reservation process on the backend. (POST Request)',
```

Comment:
I guess this also goes for this file, you don't have to do it within this PR, but it should be moved to mutations.